docs/sts.md

```diff
@@ -45,19 +45,28 @@ The token is a projected ServiceAccount into the Pod, and is short lived for an
    ``` 
 6. Create install manifests
    ```
-   ./openshift-install create manifests   
+   ./openshift-install create manifests
    ```
-7. Create AWS resources using the [ccoctl](ccoctl.md) tool (you will need aws credentials with sufficient permissions). The command below will generate public/private ServiceAccount signing keys, create the S3 bucket, upload the OIDC config into the bucket, set up an IAM Identity Provider that trusts that bucket configuration, and create IAM Roles for each AWS CredentialsRequest extracted above. It will also dump the files needed by the installer in the `_output` directory
+7. There are two options to expose the public endpoint used by the IAM OIDC identity provider on AWS: A) public S3 Bucket (default); B) AWS CloudFront Distribution serving thumbprints stored on private S3 Bucket:
+
+    A. Create AWS resources using the [ccoctl](ccoctl.md) tool (you will need aws credentials with sufficient permissions). The command below will generate public/private ServiceAccount signing keys, create the S3 bucket, upload the OIDC config into the bucket, set up an IAM Identity Provider that trusts that bucket configuration, and create IAM Roles for each AWS CredentialsRequest extracted above. It will also dump the files needed by the installer in the `_output` directory
    ```
-   ./ccoctl aws create-all --name <aws_infra_name> --region <aws_region> --credentials-requests-dir /path/to/credreqs.yaml/downloaded/in/step/2   
+   ./ccoctl aws create-all \
+     --name <aws_infra_name> \
+     --region <aws_region> \
+     --credentials-requests-dir ./credreqs \
+     --output-dir _output/
    ```
+
+    B. Steps to create a CloudFront Distribution is available [here](./sts-private-bucket.md)
+
 8. Copy the manifests created in the step 7 and put them in the same location as install-config.yaml in the `manifests` directory
    ```
-   cp _output/manifests/* /path/to/dir/with/install-config.yaml/manifests/
```

Comment:
changed as no `./openshift-install` command is specifying the `--dir` to something like  `/path/to/dir/with/install-config.yaml`

Reply:
ACK, reverted!

Reply:
@akhil-rane I believe to keep full consise with that sentence, we need also to change the entire document following the same idea. Look at the steps 4, 5 and 6. I reverted that change in my PR as it could be outscoped, but I believe to avoid confusion and keep in sync we could adapt those topics too.

Reply:
yes, we can do that. I believe you want to have something like  `./openshift-install create install-config --dir=/path/to/dir/of /install-config`. If yes, do you want to make that change as a separate commit in this PR?

Reply:
@akhil-rane yes, I believe we can do it in one separate PR to ship the current one faster. I will keep this thread open and update it when I have the other created.

Reply:
PR opened for this topic https://github.com/openshift/cloud-credential-operator/pull/471